app/views/email_templates/new.html.slim

```diff
@@ -0,0 +1,34 @@
+h1= t_title(:new)
+
+p Sie können die folgenden Platzhalter benützen:
+ul
+  li %{Anrede}
+  li %{Name}
+  li %{feedback-link}
+  li %{assignment-title}
+p Zum Beispiel:
+p Liebe/r %{Anrede} %{Name}
+p Wir bitten um Ihre %{feedback_link} für %{assignment_title}.
```

Comment:
is this a typo?
with dash: `%{feedback-link}` & `%{assignment-title}`  line 5 + 6
with underscore: `%{feedback_link}` & `%{assignment_title}`  line 11

and shouldn't `%{assignment_title}` have also a German placeholder name? or will this be changed later anyway?



NIT: `%{Anrede}` & `%{Name}` are capitalised, whereas `%{feedback_link}` & `%{assignment_title}` are not..